sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -99,7 +101,11 @@ class MapboxMap internal constructor(
     onMapLoadErrorListener: OnMapLoadErrorListener? = null
   ) {
     initializeStyleLoad(onStyleLoaded, onMapLoadErrorListener)
-    nativeMapWeakRef.call { (this as StyleManagerInterface).styleURI = styleUri }
+    if (styleUri.isEmpty()) {
+      nativeMapWeakRef.call { (this as StyleManagerInterface).styleJSON = "{}" }
+    } else {
+      nativeMapWeakRef.call { (this as StyleManagerInterface).styleURI = styleUri }
+    }
```

Comment:
nit : 
```
nativeMapWeakRef.call { (this as StyleManagerInterface).styleJSON = if ... else ...
```

Reply:
One is `styleJSON` and the other is `styleURI`, so we can't apply the if-else block you suggested.
This is for users who want to create a style fully programmatically on the application itself. For more detail, you can refer to the linked ticket.

Reply:
Ok!